doc/dev-guide/lvmo-units.md

```diff
@@ -0,0 +1,23 @@
+## LVM Volume Groups
+
+- *lvmVG* reconcile units deploys and manages LVMVolumeGroup CRs
+- The LVMVG resource manager creates individual LVMVolumeGroup CRs for each
+  deviceClass in the LVMCluster CR. The vgmanager watches the LVMVolumeGroup
+  and creates the required volume groups on the individual nodes based on the
+  specified deviceSelector and nodeSelector.
+- The corresponding CRs forms the basis of `vgManager` unit to create volume
+  groups and create lvmd config file
```

Comment:
How is the lvmd config file generated on each node? Does the vgManager have a hostPath mounted? Could you add more details about that?

Reply:
- doc did explain about hostPath in the [initial](https://github.com/red-hat-storage/lvm-operator/pull/87/files#diff-a99023791c3c5bc7aa8c1977340936a533ee8c71aad9424dca56f093221ff57cR19-R25) version of this PR
- maybe I misunderstood that comment and removed that info from vg-manager doc

Reply:
@nbalacha do you want to see the mention of hostPath?

Reply:
Yes, I think having the hostpath information here would be useful.